src/config.js

```diff
@@ -6,3 +10,40 @@ export const HOSTNAME = process.env.HOSTNAME;
 export const PHOTON_SERVICE_NAME = process.env.PHOTON_SERVICE_NAME;
 export const NOMINATIM_SERVICE_NAME = process.env.NOMINATIM_SERVICE_NAME;
 export const FILE_SERVICE_NAME = process.env.FILE_SERVICE_NAME;
+
+// Load .env files in a similar way to the bikehopper-ui frontend
+// .env.development.local, .env.development, .env, etc
+//
+// We should probably only use this for development and rely on Docker
+// supplying the env vars in production
+
+const NODE_ENV = process.env.NODE_ENV;
+if (!NODE_ENV) {
+  throw new Error(
+    'The NODE_ENV environment variable is required but was not specified.'
+  );
+}
+
+const dotenvPath = '.env';
+
+// https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+const dotenvFiles = [
+  `${dotenvPath}.${NODE_ENV}.local`,
+  // Don't include `.env.local` for `test` environment
+  // since normally you expect tests to produce the same
+  // results for everyone
+  NODE_ENV !== 'test' && `${dotenvPath}.local`,
```

Comment:
I don think we should do this. right now we dont need the test file but in other projects i've needed a .env.test file to set env vars that are useful in unit testing. I think if such a file exists we should commit it and use it.